components/_patterns/02-molecules/menus/main-menu/_00-main-menu.scss

```diff
@@ -1,68 +1,24 @@
-// Toggle
-
-.toggle-expand {
-  @include link;
-
-  display: inline-block;
-  padding: $space;
-  text-decoration: none;
-  width: 100%;
-}
-
-.toggle-expand__text {
-  display: block;
-  font-size: 0.9rem;
-  font-weight: bold;
-
-  .menu-open & {
-    color: $white;
-  }
-}
-
-.toggle-expand__icon {
-  width: 5rem;
-  padding-bottom: 3rem;
-  margin: 0 auto;
-}
-
-.toggle-expand__open {
-  display: inline-block;
-  text-align: center;
-
-  .menu-open & {
-    display: none;
-  }
-}
-
-.toggle-expand__close {
+// Nav
+.main-nav {
   display: none;
-  text-align: center;
 
-  &::before {
-    color: $white;
-    content: "X";
+  @include large {
```

Comment:
I'd like to see component-specific breakpoints used in these files instead of the "global" breakpoints

Reply:
very interesting. I love this concept in theory - breakpoints should be based on when the specific piece of content needs to change design so this makes total sense. Want me to take a stab at this or do you have specific instructions?